include/openmc/lattice.h

```diff
@@ -234,14 +225,9 @@ class RectLattice : public Lattice
   void to_hdf5_inner(hid_t group_id) const;
 
 private:
-  std::array<int, 3> n_cells_;    //!< Number of cells along each axis
+  array<int, 3> n_cells_;         //!< Number of cells along each axis
   Position lower_left_;           //!< Global lower-left corner of the lattice
   Position pitch_;                //!< Lattice tile width along each axis
-
-  // Convenience aliases
-  int &nx {n_cells_[0]};
```

Comment:
So, yeah... This screws up device code. It's because we can construct a reference to stuff on the host (probably implemented as a pointer?), but then later run in device code. If you just copy data directly to the device, that can cause a segfault. Removing these convenience aliases doesn't add too much inconvenience since it's something like 50 extra characters total as a result of removing these.